packages/navigation_legacy/src/UIShareManager/index.js

```diff
@@ -19,14 +17,6 @@ export type ShareOptions = {
 };
 
 export default class UIShareManager {
-    // Private
-    static copyToClipboard(message: string, success: string) {
-        if (!message) {
-            return;
-        }
-        Clipboard.setString(message);
-        UIToastMessage.showMessage(success);
-    }
```

Comment:
Oh, btw. Will you also apply changes in Surf? As there are many screens which use this function (as it actually not "private" as the comment claims it to be :D)

Plus we are to make Chat & Browser support the new `onLongPressText` prop as well

Reply:
Oh, you will only need to pass the `onLongPressText` handle to both `UIChatList` & `UIBrowser` and process it for these two cases only.

In that case I would copy the text & show the notice directly from Surf (using Clipboard & UINotice instance (probably, shared is enough).